### PR TITLE
feat: gno social feeds

### PR DESCRIPTION
**Pull request by hthieu1110**

### CODE REVIEW

File: packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx [hunk 1]
```diff
@@ -306,15 +309,55 @@ export const NewsFeedInput = React.forwardRef<
...
+                msg.category.toString(),
+                msg.metadata,
+              ],
+            };
+
+            const txHash = await adenaDoContract(
+              selectedNetworkId,
+              [{ type: "/vm.m_call", value: vmCall }],
+              {
+                gasWanted: 1_000_000,
```
Comment (omniwired) [inline packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx:339]: is this OK, hardcoded?

File: packages/components/socialFeed/NewsFeed/NewsFeedQueries.ts [hunk 2]
```diff
@@ -161,17 +161,58 @@ export const createPost = async ({
...
+        msg.category.toString(),
+        msg.metadata,
+      ],
+    };
+
+    const txHash = await adenaDoContract(
+      networkId,
+      [{ type: "/vm.m_call", value: vmCall }],
+      {
+        gasWanted: 2_000_000,
```
Comment (omniwired) [inline packages/components/socialFeed/NewsFeed/NewsFeedQueries.ts:191]: ?

File: packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx (hunk 1 above)
Comment (hthieu1110) [reply packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx]: yes, for this value, default is 1_000_000 so I can remove that

File: packages/components/socialFeed/NewsFeed/NewsFeedQueries.ts (hunk 2 above)
Comment (hthieu1110) [reply packages/components/socialFeed/NewsFeed/NewsFeedQueries.ts:191]: yes, 1M does not work so I hardcode to one value that works to make user flow easier, for now I don't see another way to estimate gas efficiently

**Review by omniwired — APPROVED** [review]

File: packages/components/socialFeed/NewsFeed/NewsFeedQueries.ts (hunk 2 above)
Comment (n0izn0iz) [reply packages/components/socialFeed/NewsFeed/NewsFeedQueries.ts:191]: we can't simulate tx with gno yet

**Pull request merged**